README.md

```diff
@@ -1 +1,16 @@
-# Webpack-setup
\ No newline at end of file
+# Webpack-setup
+
```

Comment:
Normally you can expect that in `README.md` there will be instructions on how to setup and how run the application.